nav2_util/src/node_utils.cpp

```diff
@@ -75,6 +75,18 @@ std::string time_to_string(size_t len)
   return output;
 }
 
+void declare_parameter_if_not_declared(
```

Comment:
Please put this in the new `nav2_ros_common` package where the other parameter utilities are relocated